indico/modules/events/views.py

```diff
@@ -120,9 +120,43 @@ def _getHeader(self):
     def getJSFiles(self):
         return WPDecorated.getJSFiles(self) + self._asset_env['modules_event_display_js'].urls()
 
+    def _getJsonLDData(self):
+        lddata = {
+            "@context": "http://schema.org",
+            "@type": "Event",
+            "url": self.event.external_url,
+            "name": self.event.title,
+            "startDate": self.event.start_dt.isoformat(),
```

Comment:
`start_dt `(UTC) or `start_dt_local` (event TZ)? Not sure what the spec requires

Reply:
The spec wants isoformat, see http://schema.org/DateTime. But dt_local seems ok.

Reply:
sure, isoformat makes sense, the comment was just about whether to use UTC or local